src/wizard/boards.ts

```diff
@@ -157,4 +157,62 @@ export const boardSelectOptions = html`
     <option value="xinabox_cw02">XinaBox CW02</option>
     <option value="nscreen-32">YeaCreate NSCREEN-12</option>
   </optgroup>
+  <optgroup label="ESP32-C3">
+    <option value="adafruit_qtpy_esp32c3">adafruit_qtpy_esp32c3</option>
+    <option value="airm2m_core_esp32c3">airm2m_core_esp32c3</option>
+    <option value="bee_motion_mini">bee_motion_mini</option>
+    <option value="deneyapkartg">deneyapkartg</option>
+    <option value="dfrobot_beetle_esp32c3">dfrobot_beetle_esp32c3</option>
```

Comment:
We should write-out all the names. If we don't do that, we might as well replace it all with an open input text box. Should be possible with some scraping of the platformio docs